test/integration/basic_test.go

```diff
@@ -154,7 +171,7 @@ func FeatureContext(s *godog.Suite) {
 
 	s.BeforeScenario(func(interface{}) {
 		testDir := setUp()
-		os.RemoveAll(testDir)
+		fmt.Println("Running Integration test in : ", testDir)
```

Comment:
+1, eventually it would be nice to have this as some configurable log output, but let's have it as message to std out for now